modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenConstants.java

```diff
@@ -251,6 +251,9 @@
     public static final String SUPPORTS_ES6 = "supportsES6";
     public static final String SUPPORTS_ES6_DESC = "Generate code that conforms to ES6.";
 
+    public static final String NULL_SAFE_ADDITIONAL_PROPS = "nullSafeAdditionalProps";
+    public static final String NULL_SAFE_ADDITIONAL_PROPS_DESC = "Set to make additional properties types declare that their indexer may return undefined";
```

Comment:
Suggest moving it to AbstractTypescript codegen. I believe the purpose of CodegenConstants is to store stuff that is applicable to many if not all generators. While the change introduced in this PR is very typescript-specific

Reply:
Gotcha. I was following the pattern of SUPPORTS_ES6 above, but it does seem a bit too broad.

Reply:
I see! Yeah it looks inconsistent across the app. SUPPORTS_ES6 was introduced years ago, but some later addition (NPM_VERSION, etc) are made directly in AbstractTypescript, which makes more sense to me.